vallum/src/main/kotlin/Vallum.kt

```diff
@@ -39,6 +40,7 @@ constructor(
             val accessToken = getAccessToken(client)
             client.get(baseUrl.trimEnd('/') + "/company-surveys") {
                 parameter("project", project)
+                parameter("active", active)
```

Comment:
filter by status 'active' is not available yet from the UI, isn't it?

Reply:
no this is just to exclude them when calling from AnyLogic.